packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -772,7 +772,7 @@ oie.registration.form.title = 》Šîĝñ ûþ ヸ€홝한Ӝฐโ《
 oie.registration.form.submit = 》Šîĝñ Ûþ ヸ€홝한Ӝฐโ《
 oie.user.profile.lastname = 》Ļåšţ ñåɱé ئ䀕ヸ€홝한Ӝฐโ《
 oie.user.profile.firstname = 》Ƒîŕšţ ñåɱé ⾼ئ䀕ヸ€홝한Ӝฐโ《
-oie.user.profile.primary.email = 》Þŕîɱåŕý éɱåîļ ⾼ئ䀕ヸ€홝한Ӝฐโ《
+oie.user.profile.primary.email = 》Éɱåîļ 홝한Ӝฐโ《
```

Comment:
This is not my change. It got generated when I ran the grunt task.

Reply:
It is from OKTA-394156. Not your change.

@nikhilvenkatraman-okta probably we need a guide/linting how to make sure the pseudo-loc is always in sync

Reply:
We have a lint in to keep them in sync for new keys added, but I think it does not check if you update the value for an existing key.

Reply:
Created https://oktainc.atlassian.net/browse/OKTA-398883 for tracking this